adcc/workflow.py

```diff
@@ -377,13 +411,12 @@ def estimate_n_guesses(matrix, n_states, singles_only=True):
     return max(n_states, n_guesses)
 
 
-def find_guesses(guessfctn, matrix, n_guesses, n_guesses_doubles=None):
+def obtain_guesses_by_inspection(matrix, n_guesses, kind, n_guesses_doubles=None):
```

Comment:
Well that's kind of deliberate. The idea is that unlike solving an approximation or so, one just looks at the elements ... in this case only the diagonals, but that might not necessarily be the case. Also there is already `guesses_from_diagonal`, which is the core building block to the current function, so I wanted to use some other name to avoid confusion.

If you think that's being too general in the name, I'm happy to change it.

Reply:
Okay, let's leave it as is.